wordpress-mobile/WordPress-iOS
# Create flag and update label for a conflicted post

- Create a BOOL flag for a post that has a conflict in the `dateModified` property between the local and web versions
- Update the status on the PostCard and PostCompact cells to display ‘Version Conflict` in `error` coloring, when a post has a version conflict

NOTES: 
- Android uses `isConflicted`, but I felt `hasVersionConflict` was more precise, and more future-proof (ie, there may be other types of conflicts in the future, and this leaves room for those names
- I opted to only update the local status, ala “Local Changes”, and not the post status, since this is something that is restricted to local - please let me know if I should incorporate it as a proper Post Status, and I can do that 

Fixes (in part) #13683 

To test:
1. Create a draft while I am online in the app. That’ll create the draft in remote.
2. Go offline
3. Make changes to the post and go back. This will save the changes as local draft
4. Go to web or any other client and make changes to the post.
5. Publish the post.
6. on the app, you should be on the posts list. turn airplane mode off
7. pull to refresh.
8. you should see now the post has a “Conflicted” state.

PR submission checklist:
